Zone5/Users/LoginResponse.swift

```diff
@@ -32,5 +32,15 @@ public struct LoginResponse: Codable {
 	
 	public var refresh: String?
 	
+	public var features: Int? // bitmask featureset
```

Comment:
The bitmasks all tend to get cases added regularly on the server side. It is why they are done as bitmasks. We had some json decoding issues a while ago with this library with unknown enum cases. It seemed safer for a value that I doubt anyone will look at anyway. Plus I personally have no idea what these feature cases are so would have to go bug someone to find out and I didn't deem it important enough. I tossed up leaving it out, but it is present in the response from the server so I opted to add it just in case someone wanted it.

Reply:
It's worth noting that you don't typically use enums with `OptionSet`, they're usually `RawRepresentable` structs that have predefined static cases defined. The `OptionSet` protocol adds a bunch of functionality to allow them to be inspected without having to do actual bitmasking.